### DISCUSSION TITLE
`ftd init` command along with a README.md

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#618] [Ideas & RFCs]
How can we provide a better experience for users of the ftd library by creating an ftd init command that generates a boilerplate code and a Read.ME file to guide users on how to organize and write their code effectively?

[Reposting from discord]
Asked by @saurabh-lohiya

**Comment by amitu** [comment]
@gargajit, @AthulA10 we already have a similar command: https://fastn.io/create-fastn-package/. Maybe we can rename it to `fastn init` as that is smaller. We also do not generate README file, which is also a good idea. 

Ideally we want to make it a template based, so you can pass any template repository, eg `fastn init -t fastn/fasnt-template` and it clones and uses that template directory as starting point.

**Comment by saurabh-lohiya** [comment]
Not just `READ.me` but also folders like `assets` and `components` so that the naming convention becomes standard and code becomes more organized

**Reply by amitu** [reply]
This is also a good idea.
